lib/truncate.coffee

```diff
@@ -0,0 +1,8 @@
+{ take } = require 'underscore'
+
+module.exports = (text, length = 50) ->
+  return unless text
+  tokens = text.split ' '
+  truncated = take(tokens, length).join(' ')
+  truncated = truncated + "..." if tokens.length > length
+  truncated
```

Comment:
`{ truncate } = require 'underscore.string'`

Reply:
Our truncate ensures that words don't get cut off, underscore.string does it by the character.

Reply:
works for me

Reply:
but i still dunno why you writing coffeescript these days

Reply:
Cause I'm a maniac

Reply:
This was actually already written, I just put it in its own file.

Reply:
tru